models/buildReport.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+const Joi = require('joi');
+// const mutate = require('../lib/mutate');
+
+const MODEL = {
+    buildId: Joi
+        .number().integer().positive()
+        .description('Build identifier')
+        .example(123345),
+
+    jobId: Joi
```

Comment:
If fields here are the same as the build, event, step tables, we should read from those files instead of redefining it here

Reply:
I think what Min meant is can write it like:
`jobId: Joi.reach(models.job.base, 'id')` instead of redefining

Reply:
makes sense, misunderstood, will update

Reply:
updated